airflow/providers/http/operators/http.py

```diff
@@ -84,6 +88,7 @@ def __init__(
         extra_options: Optional[Dict[str, Any]] = None,
         http_conn_id: str = 'http_default',
         log_response: bool = False,
+        auth_type: Any = HTTPBasicAuth,
```

Comment:
```suggestion
        auth_type: Type[AuthBase] = HTTPBasicAuth,
```

See https://docs.python-requests.org/en/master/user/authentication/#new-forms-of-authentication

Reply:
Woops done.